### PR TITLE
Updating readme.md

### PR DESCRIPTION
**Pull request by Tara-Q**
null

-----
[View rendered README.md](https://github.com/CodeYourFuture/scot-callback-cats/blob/read-me-editing/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,39 +16,27 @@ Mac users:
    ```
    psql -d cyf-bike-app-dev -f server/scripts/create_database.sql
    ```
+## Description
+We developed a tool to help a local charity, Bikes for Refugees, more efficiently schedule bike collections for their clients. Built using React (with Boostrap and vanilla CSS), Node.js and Express, and Postgresql for the database. This was our final project for the Code Your Future Full Stack Web Development course, of which we were in Scotland Class 5. We had four weeks to design and build our product and present it to an external audience.
```
Comment (bonboh) [inline README.md:20]:
```suggestion
We developed a tool to help a local charity, Bikes for Refugees, more efficiently schedule bike collections for their clients. Built using React (with Bootstrap and vanilla CSS), Node.js and Express, and Postgresql for the database. This was our final project for the Code Your Future Full Stack Web Development course, of which we were in Scotland Class 5. We had four weeks to design and build our product and present it to an external audience. 
```
Typo

File: README.md [hunk 2]
```diff
@@ -16,39 +16,27 @@ Mac users:
...
-- [x] Production build (single deployment artifact, React loaded via CDN)
-- [x] [Heroku] deployment
-- [x] [Cloud Foundry] deployment
-- [x] [Docker] build
-- [x] [Postgres] database with [node-postgres]
+## Main Features 
+Main features include:
+- A waiting list page with user details and booking statuses, retrieved from a Postgresql database
+- Ability to select users from the list and send a customisable message
+- The message UUID is stored in the database
```
Comment (bonboh) [inline README.md:29]:
```suggestion
```
We can probably remove this as it's not a feature you'd describe to users of the app. It's more of a code implementation detail

**Review by bonboh — CHANGES_REQUESTED** [review]
Hope you don't mind me popping in and giving an unsolicited review 😉 

- Can you move the postgres dev setup to before the scripts section? Coding instructions usually come after descriptions
- Can you add your names in the contributors section? It makes it feel more human to see names rather than just pasted links. You could also list them in bullet points
- Can we remove the debugging section? That's instructions for CYF trainees but is not necessary for open source projects

**Comment by Tara-Q** [comment]
> Hope you don't mind me popping in and giving an unsolicited review 😉
> 


PLEASE DO, I just didn't want to tag you in case you were busy

File: README.md (hunk 1 above)
Comment (Tara-Q) [reply README.md:20]: 👻

**Review by bonboh — APPROVED** [review]
LGTM!

**Pull request merged**